apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -127,11 +127,12 @@ export class CustomRushAction extends BaseRushAction {
 
   protected onDefineParameters(): void {
     if (this._isParallelized()) {
-      this._parallelismParameter = this.defineIntegerParameter({
+      this._parallelismParameter = this.defineStringParameter({
         parameterLongName: '--parallelism',
         parameterShortName: '-p',
         key: 'COUNT',
-        description: 'Specify the number of concurrent build processes.'
+        description: 'Specify the number of concurrent build processes'
+          + ` A value of 'max' can be used to achieve the maximum default parallelism.`
           + ' If omitted, the parallelism will be based on the number of CPU cores.'
```

Comment:
Maybe clarify:

'A value of "max" will use the number of CPU cores.  If omitted, the default parallelism will use most but not all CPU cores."